src/library_wasmfs_jsimpl.js

```diff
@@ -53,6 +53,7 @@ addToLibrary({
     return wasmFS$backends[backend].getSize(file);
   },
 
+  _wasmfs_jsimpl_set_size__i53abi: true,
```

Comment:
@sbc100 this is not for pointers, but because `off_t` is an i64, so `size` becomes a bigint here, which fails in the implementation in library_wasmfs_js_file.js where it tries to create a new Uint8Array from it. The other functions here all have buffer args along with the offset args, so they are already using the i53 abi. Is this a good solution? Are there other places it might be needed (i.e. where we might want the i53 ABI for non-pointers)?

Reply:
No, i53abi means that `off_t` and `int64_t` become i53 numbers.  It explicitly avoids bigint or i32 pairs.

The times you want i53 abi is basically any time you know that the range of an i64 will be within i53 range.

Reply:
Sorry I misread your comment.  I should have answers in agreement.